src/main/java/Task.java

```diff
@@ -0,0 +1,41 @@
+public class Task {
+    public enum TaskType {TODO, EVENT, DEADLINE, DEFAULT};
+    private TaskType taskType;
+    private String description;
+    private boolean done;
```

Comment:
Maybe make your variable sound like boolean?
```suggestion
    private boolean isDone;
```